libraries/push/impl/src/main/kotlin/io/element/android/libraries/push/impl/pushgateway/PushGatewayNotifyRequest.kt

```diff
@@ -42,9 +47,12 @@ class DefaultPushGatewayNotifyRequest(
                     devices = listOf(
                         PushGatewayDevice(
                             params.appId,
-                            params.pushKey
+                            params.pushKey,
+                            PusherData(mapOf(
+                                "cs" to secret,
+                            ))
```

Comment:
Please use a FAKE_SECRET, it will be simpler (nearly all the change in this file can be reverted) and the secret will not be used anyway when the push will be received. It will also enforce the fact that there is no need to have a session to perform this test.